packit.yaml

```
@@ -0,0 +1,17 @@
+specfile_path: pykickstart.spec.in
+synced_files:
+  - packit.yaml
+  - pykickstart.spec.in
+jobs:
+  - trigger: release
+    release_to:
+      - master
+  - trigger: pull_request
+    release_to:
+      - master
+upstream_name: pykickstart
```

```suggestion
upstream_project_name: pykickstart
```
we have decided to rename these two keys to be more descriptive, the change will land in packit 0.2 which we release this week